examples/heatTransfer/read/heatRead_adios2.cpp

```diff
@@ -109,7 +111,7 @@ int main(int argc, char *argv[])
     double *T = new double[vT->m_AvailableSteps * readsize[0] * readsize[1]];
```

Comment:
adios2::Dims is already std::vector<size_t>, just change std::vector<uint64_t> to adios2::Dims, no need to create a separate dimension vector.